pkg/server/settingswatcher/settings_watcher.go

```diff
@@ -439,3 +443,33 @@ func (s *SettingsWatcher) GetStorageClusterVersion() clusterversion.ClusterVersi
 	}
 	return s.mu.storageClusterVersion
 }
+
+// GetFn is a function to retrieve data from the kv store. It is implemented
+// by both (*kv.DB).Get and (*kv.Txn).Get.
+type GetFn = func(context.Context, interface{}) (kv.KeyValue, error)
```

Comment:
Given what we now know, maybe we just change this to take a `*kv.Txn` or some interface called `Txn` which just has this `Get` method if you don't want to depend on the concrete impl.